components/layout/Header.jsx

```diff
@@ -76,7 +87,10 @@ export default function Header() {
                 alignItems: 'center',
               }}
             >
-              <NavigationLogo fullSize={matches} />
+              <NavigationLogo 
+                fullSize={matches} 
+                data-testid="MenuIcon"
+                className="hamburger-btn" />
```

Comment:
Eslint errors here as well. Hitting save should fix. I am not sure why there are a couple more on things you didn't change though.